packages/formation/sass/modules/_m-megamenu.scss

```diff
@@ -90,7 +93,7 @@ $marketing-container-height: 380px;
       white-space: normal;
 
       h3 {
-        display: initial;
+        display: block;
```

Comment:
This was changed so the headings would appear in IE11. They were continuing to hold onto the `display: none` declaration.